src/util/Recoil_batcher.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @emails oncall+recoil
+ * @flow strict
+ * @format
+ *
+ * This is a stub for some integration into FB internal stuff
+ */
+
+const {unstable_batchedUpdates} = require('ReactDOM');
```

Comment:
I believe you meant this
```suggestion
const {unstable_batchedUpdates} = require('./Recoil_ReactBatchedUpdates');
```

Reply:
Fixed, thanks.